aws/logs_monitoring/tests/test_parsing.py

```diff
@@ -924,32 +724,8 @@ def test_awslogs_handler_step_functions_tags_added_properly(
         context = None
         metadata = {"ddsource": "postgresql", "ddtags": "env:dev"}
 
-        self.assertEqual(
-            [
-                {
-                    "aws": {
-                        "awslogs": {
-                            "logGroup": "/aws/vendedlogs/states/logs-to-traces-sequential-Logs",
-                            "logStream": "states/logs-to-traces-sequential/2022-11-10-15-50/7851b2d9",
-                            "owner": "425362996713",
-                        }
-                    },
-                    "id": "37199773595581154154810589279545129148442535997644275712",
-                    "message": '{"id":"1","type":"ExecutionStarted","details":{"input":"{"Comment": "Insert your JSON here"}","inputDetails":{"truncated":false},"roleArn":"arn:aws:iam::425362996713:role/service-role/StepFunctions-logs-to-traces-sequential-role-ccd69c03"},",previous_event_id":"0","event_timestamp":"1668095539607","execution_arn":"arn:aws:states:sa-east-1:425362996713:express:logs-to-traces-sequential:d0dbefd8-a0f6-b402-da4c-f4863def7456:7fa0cfbe-be28-4a20-9875-73c37f5dc39e"}',
-                    "timestamp": 1668095539607,
-                }
-            ],
-            list(awslogs_handler(event, context, metadata)),
-        )
-        self.assertEqual(
-            {
-                "ddsource": "stepfunction",
-                "ddtags": "env:dev,test_tag_key:test_tag_value",
-                "host": "/aws/vendedlogs/states/logs-to-traces-sequential-Logs",
-                "service": "stepfunction",
-            },
-            metadata,
-        )
+        verify_as_json(list(awslogs_handler(event, context, metadata)))
+        verify_as_json(metadata, options=NamerFactory.with_parameters("metadata"))
```

Comment:
@ge0Aja that's a useful pattern when we really want to approve 2 different things in the same test. Ideally there would be only one verify/assert but this is acceptable.